check

```diff
@@ -412,7 +402,7 @@ usage() {
       # additional info
       echo >&${out_descriptor}
```

Comment:
is there any purpose to prefer using `echo` instead of `msg` (defined in this file)?